cuda/src/utils/cuda_error_handling.hpp

```diff
@@ -6,32 +6,26 @@
  */
 #pragma once
 
+#include <cuda_runtime_api.h>
+
 // Detray Data Model include(s).
 #include "vecmem/utils/types.hpp"
```

Comment:
This include is no longer necessary...

Reply:
The macro `VECMEM_CUDA_ERROR_CHECK` uses some CUDA runtime definitions like `errorCode` and `cudaSuccess`, so I think it's a good idea to keep this import.